Assets/MixedRealityToolkit/Definitions/InputSystem/PointerOption.cs

```diff
@@ -19,23 +19,23 @@ public struct PointerOption
         /// <param name="controllerType"></param>
         /// <param name="handedness"></param>
         /// <param name="pointerPrefab"></param>
-        public PointerOption(SupportedControllerType controllerType, Handedness handedness, GameObject pointerPrefab)
+        public PointerOption(Type controllerType, Handedness handedness, GameObject pointerPrefab)
         {
             this.controllerType = controllerType;
             this.handedness = handedness;
             this.pointerPrefab = pointerPrefab;
         }
 
-        [EnumFlags]
         [SerializeField]
         [Tooltip("The controller this pointer will attach itself to at runtime.")]
-        private SupportedControllerType controllerType;
+        [Implements(typeof(IMixedRealityController), TypeGrouping.ByNamespaceFlat)]
+        private SystemType controllerType;
 
         /// <summary>
         /// The type of Controller this pointer will attach itself to at runtime.
         /// </summary>
         /// <remarks>If <see cref="Microsoft.MixedReality.Toolkit.Utilities.Handedness.None"/> is selected, then it will attach to any controller type</remarks>
-        public SupportedControllerType ControllerType => controllerType;
+        public Type ControllerType => controllerType;
```

Comment:
This should by `SystemType` as well to match the backing field.

Reply:
Likely the same goes for all the usages of this property as well.

Reply:
working on it ;)

Reply:
done